### ISSUE TITLE
pip installation returning error using docker file

### ISSUE THREAD

**Issue by karandogra987** [legolason/PyIICCF#2]
pip installation and up-gradation or any other command using pip is returning a syntax error. I have attached a screenshot of the error and the command-line which returns
![pip_error](https://github.com/legolason/PyIICCF/assets/152680673/f667f21d-2e73-4324-92cb-26477c1d6b04)
 error.

PS: I am doing the PYIICCF installation using the docker file.

**Comment by karandogra987** [comment]
I have tried to use other commands for installing or upgrading pip, but none has worked so far.

**Comment by legolason** [comment]
Yes, install this package is tough, especially for carma_pack, the most common error is the wrong directory. Also this is only for Python 2. You can try to install it step by step.
